.github/workflows/build.yml

```diff
@@ -0,0 +1,22 @@
+name: build
+on: [pull_request, push]
+
+permissions:
+  contents: read # to fetch code (actions/checkout)
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.8', '3.9', '3.10', '3.11']
+    steps:
+      - uses: actions/checkout@v3
+      - run: |
+           docker build -f docker/Dockerfile \
+             --build-arg PYTHON_VERSION=${{ matrix.python-version }} \
+             --tag metaworld-docker .
+      - name: Run tests
+        run: docker run metaworld-docker pytest tests/*
+#      - name: Run doctests
```

Comment:
What does this do exactly? Should we have doc tests for other projects?

Reply:
Did some googling looks useful, though i don’t think many other farama projects do that style of docs with code examples

Reply:
This was a copy from Gymnasium where we run this. 
I recommend it for having docstrings that are correct and will raise an error if the code changes etc